shell/platform/android/hardware_buffer_external_texture.cc

```diff
@@ -39,7 +39,8 @@ void HardwareBufferExternalTexture::Paint(PaintContext& context,
         flutter::DlCanvas::SrcRectConstraint::kStrict  // enforce edges
     );
   } else {
-    FML_LOG(WARNING) << "No DlImage available.";
+    FML_LOG(WARNING)
+        << "No DlImage available for HardwareBufferExternalTexture to paint.";
```

Comment:
What's the developer action to take here?

If there is none, maybe this should be a DLOG/DCHECK?

Reply:
This is useful information for debugging the engine and I'd like to keep it in place for now until this code is more solid.